fm.reaper.Reaper.yml

```
@@ -18,15 +18,15 @@ finish-args:
 add-extensions:
   org.freedesktop.LinuxAudio.Plugins:
     directory: extensions/Plugins
-    version: "23.08"
+    version: "24.08"
     add-ld-path: lib
-    merge-dirs: lv2;vst;vst3
+    merge-dirs: ladspa;dssi;lv2;clap;vst;vst3
```

does reaper support these formats or did you just cut & paste?

---

no. this has nothing to do with the base extension either.

```suggestion
    merge-dirs: lv2;clap;vst;vst3
```

this assuming the previous was correct.